output.txt

```diff
@@ -0,0 +1,8 @@
+You are now connected to database "nc_news_test" as user "hannahreed".
+ slug  |          description           
+-------+--------------------------------
+ mitch | The man, the Mitch, the legend
+ cats  | Not dogs
```

Comment:
As this is for your own benefit, I probably wouldn't include it in any PRs (absolutely fine for you to work with stuff like this locally, though)